### PR TITLE
adding discovery

### PR DESCRIPTION
**Pull request by sileod**

### CODE REVIEW

File: datasets/discovery/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+annotations_creators:
+- other
+language_creators:
+- other
+languages:
+- en
+licenses: []
```
Comment (lhoestq) [inline datasets/discovery/README.md:8]: is there no license or is it unknown ?
if unknown you can use
```
licenses:
- unknown
```

File: datasets/discovery/discovery.py [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+}
+"""
+
+_Discovery_DESCRIPTION = """\
+Discourse marker prediction with 174 different markers
+"""
+
+DATA_URL = "https://www.dropbox.com/s/aox84z90nyyuikz/discovery.zip?dl=1"
```
Comment (lhoestq) [inline datasets/discovery/discovery.py:51]: where does this url come from ?

Comment (lhoestq) [reply datasets/discovery/discovery.py:51]: also is there a webpage associated with the dataset somewhere ?

**Review by lhoestq — COMMENTED** [review]
Looks very cool ! thanks fo radding this one

I left a few comments

File: datasets/discovery/discovery.py (hunk 2 above)
Comment (sileod) [reply datasets/discovery/discovery.py:51]: > where does this url come from ?

I'm the author of that dataset, I'll add my github link

File: datasets/discovery/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+size_categories:
+  discovery:
+  - n>1M
+  discoverysmall:
+  - 10K<n<100K
+source_datasets: []
+task_categories:
+- text-classification
+task_ids: []
+---
```
Comment (lhoestq) [inline datasets/discovery/README.md:20]: could you add the other sections of the dataset card ?

You can leave their content with `[More Information Needed]`

File: datasets/discovery/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+annotations_creators:
+- other
+language_creators:
+- other
+languages:
+- en
+licenses: unknown
```
Comment (lhoestq) [inline datasets/discovery/README.md:8]:
```suggestion
licenses:
- unknown
```

File: datasets/discovery/README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+multilinguality:
+- monolingual
+size_categories:
+  discovery:
+  - n>1M
+  discoverysmall:
+  - 10K<n<100K
+source_datasets: []
```
Comment (lhoestq) [inline datasets/discovery/README.md:16]:
```suggestion
source_datasets:
- original
```

File: datasets/discovery/README.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+task_categories:
+- text-classification
+task_ids: []
```
Comment (lhoestq) [inline datasets/discovery/README.md:19]:
```suggestion
task_ids:
- text-classification-other-discourse-marker-prediction
```

**Review by lhoestq — COMMENTED** [review]
Thanks for adding a link to the github repo of this dataset :) 

Before we merge, can you just take a look at the suggestions I just added about the dataset card please ?

**Review by lhoestq — APPROVED** [review]
Awesome thanks :)

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Pull request merged**